pkg/graph/schema.graphql

```diff
@@ -757,7 +782,7 @@ type Query {
     after: String
     first: Int!
   ): AccessibilityRequestsConnection
-  systemIntake(id: UUID!): SystemIntake @hasRole(role: EASI_GOVTEAM)
+  systemIntake(id: UUID!): SystemIntake
```

Comment:
I know you wanted to open this up to the business owner, but I'm curious where we're doing the security check now. I don't see additive code elsewhere.

Reply:
... check again! in the resolver, I added an auth check.